WordPress/Classes/ViewRelated/Plugins/PluginViewController.swift

```diff
@@ -88,4 +96,11 @@ class PluginViewController: UITableViewController {
         tableView.endUpdates()
     }
 
+    override func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        // This is a hack/work-around to remove the gap from the top of the tableView — the system leaves
+        // a gap with a `grouped` style by default — we want the banner up top, without any gaps.
+        guard section != 0 else { return 0 }
+
+        return 17.5
+    }
```

Comment:
Please move the magic numbers to a `Constants` enum, if needed (as static vars!).

Reason for having that as an `enum` instead of a `struct` is that: unless you add a case, you won't be able to accidentally instantiate it.